components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__genes.xml

```diff
@@ -137,10 +137,12 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
+        if (isSuggestSelected) {
```

Comment:
Agreed. This is exactly why `fetchGeneDetails()` function that generates them is now called only `if (XWiki.contextaction == "edit")`, [lines 171-173].